backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/businessentities/storage/Image.java

```diff
@@ -25,6 +25,8 @@ public class Image implements BusinessEntityWithStatus<Guid, ImageStatus> {
 
     private Date lastModified;
 
+    private Guid parentDiskId;
```

Comment:
2 thoughts, @ahadas what do you think?
1) `parentDiskId` -- > `parentImageId` (both the data member and getter/setter.
2) Cal them via `DiskImage` object by implementing something similar to the existing:
```
    public Guid getParentId() {
        return getImage().getParentId();
    }

    public void setParentId(Guid parentId) {
        getImage().setParentId(parentId);
    }
```
That way similar to `imageWithMissingParent.getParentId()` it will be possible to call
`diskImage.getParentDiskId()` (or `diskImage.getParentImageId()` if the above suggestion is accepted), without the `getImage()` in the middle, which is similar to the existing code.

Reply:
Seems like a good idea to me.